Assets/MixedRealityToolkit.SDK/Inspectors/Dwell/DwellProfileWithDecayInspector.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell.Editor
```

Comment:
>Microsoft.MixedReality.Toolkit.Experimental.Dwell.Editor [](start = 10, length = 56)

With this inspector, it would probably be also good to have this in the MRTK.Examples folder (because the thing it inspects is going to be moved to the MRTK examples folder as well)

Reply:
Sure. When i move it under Examples, I get build error though. 
error CS0115: 'DwellProfileWithDecayInspector.OnInspectorGUI()': no suitable method found to override

Seems like i need to  add it to some exclusion list in build steps. Any pointers on where that needs to be done ?

Reply:
Do you have a local branch that I could check out that has your changes in that state?

Reply:
https://github.com/ritijain/MixedRealityToolkit-Unity/tree/gazeDwellv1